python/grass/jupyter/utils.py

```diff
@@ -69,9 +71,16 @@ def reproject_region(region, from_proj, to_proj):
         raise RuntimeError(
             _("Encountered error while running m.proj: {}").format(stderr)
         )
-    enws = gs.decode(proj_output).split(os.linesep)
-    elon, nlat, unused = enws[0].split(" ")
-    wlon, slat, unused = enws[1].split(" ")
+    enws = gs.decode(proj_output).splitlines()
+    # get the largest bbox
+    n = []
+    e = []
+    for each in enws:
+        elon, nlat, unused = each.split(" ")
+        e.append(float(elon))
+        n.append(float(nlat))
+    elon, wlon = max(e), min(e)
+    nlat, slat = max(n), min(n)
```

Comment:
Maybe actually worse than e and s are elon (Elon) and slat (narrow strip or bar, leading-edge slat), but that's existing code, so possibly out of this PR.